### PR TITLE
Czech translation of Typora

### PR DESCRIPTION
**Pull request by byDave251**
I just made translation to Czech. All the translations are in folder cs-CZ.lproj.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Translating Typora
+This is Czech translation.
```
Comment (abnerlee) [inline Readme.md]: You don't need to submit this `Readme.md`, since it will also be merged into this repo

Comment (byDave251) [reply Readme.md]: How long it will take until you implement this pull request?

Comment (abnerlee) [reply Readme.md]: @byDave251 could you exclude this `Readme.md` from this MR?

Comment (byDave251) [reply Readme.md]: Excluded

**Pull request merged**